pupa/scrape/schemas/person.py

```diff
@@ -16,6 +16,7 @@
         "national_identity": { "type": "string", "blank": True },
         "contact_details": contact_details,
         "links": links,
+        "source_identified": { "type": ["null", "boolean"] },
```

Comment:
this property is used for people and orgs that when we often don't have enough information about them to disambiguate them. in other words, it keeps us from saying that "James Smith" in record #1234 is the same as "James Smith" in #7890.  that's useful in lobbying data, for instance, since a first and last name is all we get for many lobbyists.

if that was set on a per-jurisdiction basis, would it require that there be alternate jurisdictions for data that has this problem?